File: Medical_Profile/MPC_User.vb

Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks


Friend Class Blk_entry
  Public Property State As Integer
  Public Property Num As Integer
  Public Property Gb As GroupBox
End Class

#If DEBUG Then
Friend Class MPC_User
  Public Property User_Number As Integer
  Public Property Mkey As String
  Public Property Token As String
  Public Property Disk_Label As String
  Public Property Practice As String
  Public Property Secret1 As String
  Public Property Secret2 As String
  Public Property K1 As String
  Public Property K2 As String
  Public Property cid As String
  Public Property Salt As String
  Public Property Url As String
  Public Property Fname As String
  Public Property APIVersion As String
  Public Property Ftime As Long
  Public Property Email As String
  Public Property Iterations As Integer
  Public Property Phone As String
  Public Property Lines As String
  Public Property Blocks As String
  Public Property Sec_visible As String
  Public Property Sptitle As String
  Public Property Points As String
  Public Property Labels As String
  Public Property Comment As String
  Public Property Version As String
  Public Property Minimum_blocks As String
  Public Property Expiration As Date
  Public Property Blocklist As List(Of String)
End Class
#End If

Public Class MPC_key
 Public Property Mkey As String
 Public Property Email As String
 Public Property K1 As String
 Public Property Secret As String
 Public Property Dlab As String
 Public Property Url As String
 Public Property Salt As String
 Public Property Iterations As Integer
 Public Property Lines As String
 Public Property Blocks As String
 Public Property Sec_visible As Boolean
 Public Property Sptitle As String
 Public Property Points As String
 Public Property Labels As String
 Public Property Version As String
 Public Property Minimum_blocks As String
 Public Property Blocklist As List(Of String)
End Class

Friend Class MPC_type
  Public Property Akey As String
  Public Property F1 As String
End Class

File: Medical_Profile/MPC_User.cs
using global::System;
using global::System.Collections.Generic;
using System.Windows.Forms;

namespace Medical_Profile
{
 internal class Blk_entry
 {
 public string header { get; set; }
  public int State { get; set; }
  public int Num { get; set; }
 }

 /* TODO ERROR: Skipped IfDirectiveTrivia */
 internal class MPC_User
 {
  public int User_Number { get; set; }
  public string Mkey { get; set; }
  public string Token { get; set; }
  public string Disk_Label { get; set; }
  public string Practice { get; set; }
  public string Secret1 { get; set; }
  public string Secret2 { get; set; }
  public string K1 { get; set; }
  public string K2 { get; set; }
  public string cid { get; set; }
  public string Salt { get; set; }
  public string Url { get; set; }
  public string Fname { get; set; }
  public string APIVersion { get; set; }
  public long Ftime { get; set; }
  public string Email { get; set; }
  public int Iterations { get; set; }
  public string Phone { get; set; }
//  public string Lines { get; set; }
//  public string Blocks { get; set; }
  public string Sec_visible { get; set; }
  public string Sptitle { get; set; }
//  public string Points { get; set; }
//  public string Labels { get; set; }
  public string Comment { get; set; }
  public string Version { get; set; }
  public string Minimum_blocks { get; set; }
  public DateTime Expiration { get; set; }
  public List<string> Blocklist { get; set; }
 }
 
 public class MPC_key
 {
  public string Mkey { get; set; }
  public string Email { get; set; }
  public string K1 { get; set; }
  public string Secret { get; set; }
  public string Dlab { get; set; }
  public string Url { get; set; }
  public string Salt { get; set; }
  public int Iterations { get; set; }
  public string Lines { get; set; }
  public string Blocks { get; set; }
  public bool Sec_visible { get; set; }
  public string Sptitle { get; set; }
  public string Points { get; set; }
  public string Labels { get; set; }
  public string Version { get; set; }
  public string Minimum_blocks { get; set; }
  public List<string> Blocklist { get; set; }
 }

 internal class MPC_type
 {
  public string Akey { get; set; }
  public string F1 { get; set; }
 }
}